exercises/two-fer/two-fer-tests.factor

```diff
@@ -0,0 +1,8 @@
+USING: hello-world tools.test ;
```

Comment:
it may be better if this were `two-fer` instead of `hello-world`, otherwise there may be trouble finding the correct word to test. I experienced the following:

```
[vagrant@arch factor]$ time factor -run=exercism.testing two-fer
fatal: Not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
working directory OK: /vagrant is a user-env 
invalid config: exercises are missing implementations or tests


testing exercise: two-fer

two-fer/two-fer-tests.factor

6: { "One for you, one for me."   } [ ""      2-for-1 ] unit-test
                                                     ^
No word named “2-for-1” found in current vocabulary search path
(U) Quotation: [ c-to-factor => ]
    Word: c-to-factor
(U) Quotation: [ [ (get-catchstack) push ] dip call => (get-catchstack) pop* ]
(O) Word: command-line-startup
(O) Word: run-test-file
(O) Word: run-file
(O) Word: parse-file
(O) Word: parse-stream
(O) Word: parse-fresh
(O) Word: (parse-lines)
(O) Word: (parse-until)
(O) Word: parse-until-step
(O) Word: execute-parsing
(O) Word: POSTPONE: [
(O) Method: M\ POSTPONE: f parse-quotation
(O) Word: (parse-until)
(O) Word: parse-until-step
(O) Word: no-word
(O) Word: throw-restarts
(O) Method: M\ object throw
(U) Quotation: [
        OBJ-CURRENT-THREAD special-object error-thread set-global
        current-continuation => error-continuation set-global
        [ original-error set-global ] [ rethrow ] bi
    ]

real	0m10.784s
user	0m10.392s
sys	0m0.314s
```

Reply:
ah yes, not sure how that got past the automated tests. it'll be fixed in the next commit